src/index.js

```diff
@@ -0,0 +1,170 @@
+import './style.css';
+import Task from '../modules/task.js';
+import { setCheckboxListener, checkCompleted } from '../modules/status.js';
+
+const newTaskInput = document.querySelector('#new-task');
+const element = document.querySelector('#todolist');
+let toDoList = [];
+
+const getToDoList = () => {
+  toDoList = JSON.parse(localStorage.getItem('to_do_list'));
+  if (toDoList === null) {
+    return [];
+  }
+  return toDoList;
+};
+
+// eslint-disable-next-line consistent-return
+const printList = () => {
+  const list = getToDoList();
+  let tasks = '';
+  try {
+    for (let i = 1; i <= list.length; i += 1) {
+      // eslint-disable-next-line no-loop-func
+      list.forEach((task) => {
```

Comment:
While you did a good Job working on the project requirements. i noticed you disabled eslint it is against microverse policy to disable linters because linters help us write clean code. it would be nice if you enabled eslint linters.